docs/connected-accounts.rst

```diff
@@ -66,6 +65,38 @@ Repository status (``repo:status``)
     but there isn't a more granular permission
     that only allows setting up SSH keys for read access.
 
+.. _github-permission-troubleshooting:
+
+GitHub permission troubleshooting
+`````````````````````````````````
+
+**Repositories not in your list to import**.
+
+Many organizations require approval for each OAuth application that is used,
+or you might have disabled it in the past for your personal account.
+This can happen at the personal or organization level,
+depending on where the project you are trying to access has permissions from.
+
+.. tabs::
+
+   .. tab:: Personal Account
+
+       You need to make sure that you have granted access to the Read the Docs `OAuth App`_ to your **personal GitHub account**.
+       If you do not see Read the Docs in the `OAuth App`_ settings, you might need to disconnect and reconnect the GitHub service.
+
+       .. seealso:: GitHub docs on `requesting access to your personal OAuth`_ for step-by-step instructions.
+
+       .. _OAuth App: https://github.com/settings/applications
+       .. _requesting access to your personal OAuth: https://docs.github.com/en/organizations/restricting-access-to-your-organizations-data/approving-oauth-apps-for-your-organization
+
+   .. tab:: Organization Account
+
+       You need to make sure that you have granted access to the Read the Docs OAuth App to your **organization GitHub account**.
+       If you don't see |com_brand| listed, then you might need to connect GitHub in your social accounts as noted above.
```

Comment:
Think this applies for both, .org and .com?

Reply:
Perhaps, is there a specific suggestion to improve it?

Reply:
Perhaps, is there a specific suggestion to improve it?

Reply:
```suggestion
       If you don't see "Read the Docs" listed, then you might need to connect GitHub to your social accounts as noted above.
```